addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -128,6 +129,40 @@ const BUILTIN_PROPS: Dictionary = {
 	},
 }
 
+static var block_definition_dictionary: Dictionary
```

Comment:
There is already a dictionary of block definitions in https://github.com/endlessm/godot-block-coding/blob/b11b95ec28e99e0e3e0113501701780958307da6/addons/block_code/code_generation/blocks_catalog.gd#L6 . That should be used instead.

Reply:
This and another cache that was added in BlockCanvas were removed. Now pretty much everything BlockDefinition related happens in BlocksCatalog.